src/components/common/Header.tsx

```diff
@@ -21,15 +21,21 @@ const HeaderWrapper = styled.header`
   padding: 20px 120px;
   width: 100%;
   border-bottom: 1px solid ${({ theme }) => theme.colors.MAIN_FONT};
+  background: ${({ theme }) => theme.colors.BACKGROUND};
 `;
 
 const Logo = styled.h1`
   font-family: 'Bebas Neue', cursive;
   font-size: 4.8rem;
+  color: ${({ theme }) => theme.colors.MAIN_FONT};
 `;
 
 const ButtonsWrapper = styled.div`
   display: flex;
   align-items: center;
   gap: 20px;
+  /* TODO: this code is temp */
```

Comment:
```suggestion
/** @todos: this code is temp */
```

혹시 요걸 의도하신건가 싶어서 ㅋㅋ 저는 이렇게 하니까 extension이 자동으로 issue 만들어줄까? 하더라고요 ㅋㅋ

Reply:
아.. 저는 그 todo highlight를 써서 그냥 TODO로만 주로 쓰는데 해당 extension도 살펴보겠슴당

Reply:
오홍,, 그건 뭐지 찾아볼게용 [이게](https://vscode.tistory.com/entry/TODO-Highlight) 훨씬 좋은 거 같네요 근데 ㅋㅋ